apps/ledger-live-mobile/src/newArch/features/Market/utils/index.ts

```diff
@@ -92,9 +98,13 @@ export const counterValueFormatter = ({
     return formattedNumber;
   }
 
-  // FIXME: HOW DID THIS WORK WHEN CURRENCY IS EMTPY
-  // PLEASE FIX
-  return formatter ? formatter.format(value) : value + "";
+  if (formatter) {
+    const formattedValue = formatter.format(value);
+    const upperCaseTicker = ticker?.trim()?.toLocaleUpperCase() || "";
```

Comment:
```suggestion
    const upperCaseTicker = ticker?.trim()?.toLocaleUpperCase() || "";
    const upperCaseTicker = ticker?.trim()?.toLocaleUpperCase();
```

Not necessary` || "";
`